readme.md

```diff
@@ -4,19 +4,25 @@ Repository of useful FFmpeg command lines for archivists!
 
 ## What is this?
 
-Project Objective: To facilitate better understanding of FFmpeg through collaborative sharing of useful scripts and detailed flag-level description of how each script works so archivists can copy-paste and produce their own scripts but also understand how and why they work.
+**Project Objective**: To facilitate better understanding of FFmpeg through collaborative sharing of useful scripts and detailed flag-level description of how each script works, so archivists can copy-paste and produce their own scripts, but also understand how and why they work.
```

Comment:
I guess we could use the same style as below with a subtitle: `#### Project Objective`